app/partials/track.html

```diff
@@ -42,6 +42,7 @@
 
 <div ng-if="track.duration || track.duration_ms" class="track-right track-time text-right">
     <p>
+        <span ng-show="playCount">{{ playCount }} plays&nbsp;&nbsp;</span>
```

Comment:
maybe this should be in it's own element outside of track duration e.g. `track-plays`